### DISCUSSION TITLE
The contact email is missing in the header wrapper above

### DISCUSSION THREAD

**Discussion by nibinglobal** [breezefront/community#15] [1. Help]
Thanks for your great theme.
The contact us information at the top is not showing .
How do you add an address in the contact us page? using template in Theme?

**Comment by vovayatsyuk** [comment]
Email is taken from _Configuration > General > Store Email Addresses > Sales Representative_.
Store address data is taken from _Configuration > General > General > Store Information_.

Additionally, you can edit cms blocks: contact_content, footer_content, header_panel_info.

**Comment by nibinglobal** [comment]
its not showing
![Screenshot 2022-09-23 at 4 24 54 PM](https://user-images.githubusercontent.com/78966425/191947295-7f911ea3-8cda-4923-a52d-b86b40e7846b.png)

**Reply by vovayatsyuk** [reply]
1. Check if header_panel_info cms block exists
2. Check if it has come content inside
3. Check config values
4. Clean cache

**Comment by nibinglobal** [comment]
Can you please tell me the correct way to install the theme.
I installed it using composer require command.
I can't see any cms blocks in my admin dashboard.

**Reply by vovayatsyuk** [reply]
You need to run the installer. It will create cms blocks and homepage.

~~~
bin/magento marketplace:package:install swissup/breeze-evolution
~~~

https://breezefront.com/installation

**Comment by nibinglobal** [comment]
Thank you
